dashboard/visualization/z_score_plot.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
+import pandas as pd
+import plotly.graph_objs as go
+
+PLOTLY_TEMPLATE = "plotly_white"
+
+
+def prepare_data_plot(df, well, col, id_pos="id_pos", range_max=None):
```

Comment:
I'd suggest renaming this function to indicate that it prepares the dataframe and not the actual plot